expressEngine/lib/service/db.js

```diff
@@ -14,6 +14,10 @@ class DbOperation {
   findOne = (collection, data = {}, projection = {}) => {
     return collection.findOne(data, projection);
   }
+
+  delete = (collection, data) => {
```

Comment:
@adilsuccessive Instead of delete, use deleteOne. As we may have a method to bulk delete

Reply:
Done with the change